build/ansible/roles/pmm-images/tasks/main.yml

```diff
@@ -162,3 +185,22 @@
     src: grafana.ini
     dest: /etc/supervisord.d/grafana.ini
     mode: 0644
+
+- name: Create backup directory
+  file:
+    path: /srv/backup
+    state: directory
+
+- name: Create a working directory for VictoriaMetrics
+  file:
+    path: /srv/victoriametrics/data
+    state: directory
+    owner: pmm
+    group: pmm
+
```

Comment:
do we really need it here?

Reply:
Frankly, I'm. not super sure. I just wanted to put here everything that has to do with provisioning the directories.

Reply:
I think provision shouldn't be included to build process at all